package.json

```diff
@@ -23,16 +23,16 @@
     "@storybook/addon-postcss": "2.0.0",
```

Comment:
could we update markdown-viewer as well? :pray: 
to ^5.2.0

Reply:
cc @mmiask

Reply:
If it doesn't break anything, then sure 👍